components/src/preact/textInput/text-input.tsx

```diff
@@ -68,32 +68,46 @@ const TextInputInner: FunctionComponent<TextInputInnerProps> = ({
     return <TextSelector lapisField={lapisField} value={value} placeholderText={placeholderText} data={data} />;
 };
 
+type SelectItem = {
+    count: number;
+    value: string;
+};
+
 const TextSelector = ({
     lapisField,
     value,
     placeholderText,
     data,
 }: TextSelectorProps & {
-    data: string[];
+    data: SelectItem[];
 }) => {
+    const initialSelectedItem = data.find((canditade) => canditade.value == value);
+
     return (
         <DownshiftCombobox
             allItems={data}
-            value={value}
+            value={initialSelectedItem}
             filterItemsByInputValue={filterByInputValue}
-            createEvent={(item: string | null) => new TextInputChangedEvent({ [lapisField]: item ?? undefined })}
-            itemToString={(item: string | undefined | null) => item ?? ''}
+            createEvent={(item: SelectItem | null) =>
+                new TextInputChangedEvent({ [lapisField]: item?.value ?? undefined })
+            }
+            itemToString={(item: SelectItem | undefined | null) => item?.value ?? ''}
             placeholderText={placeholderText}
-            formatItemInList={(item: string) => {
-                return <span>{item}</span>;
+            formatItemInList={(item: SelectItem) => {
+                return (
+                    <div className='flex gap-2'>
+                        <span>{item.value}</span>
+                        <span className='text-gray-500'>({item.count})</span>
+                    </div>
```

Comment:
I don't know whether this is actually simpler to compute for a browser (because we don't need `flex` here)
```suggestion
                    <p>
                        <span>{item.value}</span>
                        <span className='ml-2 text-gray-500'>({item.count})</span>
                    </p>
```

Reply:
We can use this here. I don't really like the margin calls to position elements in a row, because this causes issues if they are put on different rows. But I think we can change it here, since we only have two elements.